### PR TITLE
vebt-332 - add marelby hernandez and Nawar Hussein to staging spool file email list

### PR DESCRIPTION
**Pull request by GcioGregg**
vebt-332 - add marelby hernandez and Nawar Hussein to staging spool file email list

## Summary

- *This work is behind a feature toggle (flipper): NO*
- Added marelby.hernandez@va.gov and nawar.hussein@va.gov to the list of emails that receive the spool files related to forms submitted in staging for QA purposes.
- not a bug
- *(What is the solution, why is this the solution?)* N/A
- VFEP/VEBT
- *(If introducing a flipper, what is the success criteria being targeted?)* N/A

## Related issue(s)

- [vebt-332](https://jira.devops.va.gov/browse/VEBT-332)

## Testing done

- generate a spool file on staging, check with Marelby and Nawar that they received in nightly email

## What areas of the site does it impact?
nightly email of spool files on staging (no impact on production)

## Acceptance criteria

- [x]  I fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x]  No error nor warning in the console.
- [x]  Events are being sent to the appropriate logging solution
- [x]  Documentation has been updated (link to documentation)
- [x]  No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs
- [x]  Feature/bug has a monitor built into Datadog (if applicable)

### CODE REVIEW

File: config/settings.yml [hunk 1]
```diff
@@ -224,6 +224,8 @@ edu:
       - vishnhav.ashok@va.gov
       - donna.saunders@va.gov
       - ariana.adili@govcio.com
+      - marelby.hernandez@va.gov
+      - nawar.hussein@va.gov
```
Comment (rmtolmach) [inline config/settings.yml:228]: Can you update the PR description and title to reflect adding two people and not just Marelby

Comment (GcioGregg) [reply config/settings.yml:228]: Done.  Good catch, Nawar was a late addition and I forgot to add him to the description and title.  Thanks.

**Review by nfstern02 — APPROVED** [review]

File: config/settings.yml (hunk 1 above)
Comment (rmtolmach) [reply config/settings.yml:228]: thanks!

**Review by rmtolmach — APPROVED** [review]
looks good once tests pass!

**Pull request merged**